src/calendar-list/index.tsx

```diff
@@ -223,10 +224,16 @@ const CalendarList = (props: CalendarListProps & ContextProp, ref: any) => {
     return false;
   }, [currentMonth]);
 
-  const renderItem = useCallback(({item}: {item: XDate}) => {
+  const renderItem = useCallback(({item, index}: {item: XDate, index: number}) => {
     const dateString = toMarkingFormat(item);
     const [year, month] = dateString.split('-');
     const testId = `${testID}.item_${year}-${month}`;
+    let dynamicHeight;
```

Comment:
This calculation looks like something that should be optional and passed to the component, otherwise, it will break current behavior - displaying an entire calendar month - that is what most users want.

Reply:
completely agree! So I created a new commit that isolates the functionality of getting the calendar height and made wrapped by a condition that took into consideration a component prop that by default is false but can be controlled by the user - Considering that the prop comes true, we have one more condition to check the index and know that it is the first item in the calendar and only then apply the dynamic height.

Reply:
@Inbal-Tish Any update on it?